intro/fonts/index.html

```diff
@@ -78,7 +78,7 @@ <h2><a id="rendering-fonts" class="js"></a>Rendering fonts in Pygame</h1>
         <ul>
             <li><code>text</code>, which is a string with the text you want to render;</li>
             <li><code>antialias</code>, which is a boolean indicating whether you want your text to have smooth edges;</li>
-            <li><code>color</code>, which can be a <code>tuple[int, int, int]</code> or a string.</li>
+            <li><code>color</code>, which can be a <code>tuple[int, int, int]</code>, <code>tuple[int, int, int, int]</code> (4th integer is the alpha value), <code>int</code>, <code>pygame.Color</code>, or <code>str</code>.</li>
```

Comment:
One thing I'd like to see is some specificity that tuple[int, int, int] is specifically RGB format and that tuple[int, int, int, int] is specifically RGBA (you address the A part, but I don't really see any notes about the order of the other 3). You could also reference the color article that you're writing instead of trying to list them all out imo